src/cml.test.js

```diff
@@ -75,6 +75,28 @@ describe('Github tests', () => {
 
     expect(caughtErr).toBe('No commit found for SHA: invalid_sha');
   });
+
+  test('Runner logs', async () => {
+    const cml = new CML();
+    cml.driver = 'github';
+    let log = cml.parseRunnerLog({ data: 'Listening for Jobs' });
+    expect(log.status).toBe('ready');
+
+    log = cml.parseRunnerLog({ data: 'Running job' });
+    expect(log.status).toBe('job_started');
+
+    log = cml.parseRunnerLog({ data: 'completed with result: Succeded' });
```

Comment:
```suggestion
    log = cml.parseRunnerLog({ data: 'completed with result: Succeeded' });
```

See [above](https://github.com/iterative/cml/pull/689/files#r820782354)